### PR TITLE
got rid of caching and changes reads total to reads achieved closes #3

### LINKED ISSUE #3: Rename ‘Reads Total’ to ‘Reads Achieved’ or something similar

### PR DESCRIPTION
**Pull request by xshirl**

### CODE REVIEW

File: frontend/src/Table.js [hunk 1]
```diff
@@ -2,17 +2,52 @@ import React, { useState, useEffect } from 'react';
...
+function getWindowDimensions() {
+  const { innerWidth: width, innerHeight: height } = window;
+  const { clientWidth: width2, clientHeight: height2} = document.documentElement;
+  let tableHeight = height - 100;
+  return {
+    tableHeight
+  };
+}
+
+// function useWindowDimensions() {
```
Comment (dmcmanam) [inline frontend/src/Table.js:28]: Let's remove all dead code unless it is part of an open issue/TODO that will be included soon.

**Review by dmcmanam — APPROVED** [review]

File: frontend/src/Table.js [hunk 2]
```diff
@@ -35,7 +47,7 @@ function HomePage() {
   const [isLoading, setIsLoading] = useState(true);
   const [searchTerm, setSearchTerm] = React.useState('');
   const [sorting, setSorting] = React.useState(true);
-
+  const [sortedExcelData, setSortedExcelData] = React.useState([]);
```
Comment (DavidStreid) [inline frontend/src/Table.js:50]: Is this used anywhere?

File: frontend/src/App.css [hunk 3]
```diff
@@ -6,6 +6,32 @@ body {
...
-  overflow: hidden;
   font-size: 15px;
+  overflow: hidden;
+}
+
+html {
+  overflow: hidden;
+}
+
+/* .handsontable {
```
Comment (DavidStreid) [inline frontend/src/App.css:17]: Can we get rid of these commented-out sections & `console.log` statements?

**Review by DavidStreid — COMMENTED** [review]
Looks good! I just have some clean-up questions

**Pull request merged**